packages/fx-core/src/component/generator/scaffoldContext.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { LogProvider } from "@microsoft/teamsfx-api";
+import AdmZip from "adm-zip";
+import { ScaffoldAction } from "./scaffoldAction";
+
+export interface ScaffoldContext {
+  scenario: string;
+  destination: string;
+  logProvider: LogProvider;
+  zipUrl?: string;
+  zip?: AdmZip;
+  fallback?: boolean;
```

Comment:
Do we need this parameter or just check the fallback url is none

Reply:
You are right. Only fallbackUrl will suffice.